apps/perf-test-react-components/src/scenarios/FluentProviderWithTheme.tsx

```diff
@@ -5,9 +5,11 @@ import { teamsLightTheme } from '@fluentui/react-theme';
 const LayoutShift: React.FunctionComponent = ({ children }) => {
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28887&commitId=62fff81864083447b7d0fa4ac4f320be73cc9711&env=prod&clientType=FLUENTUI)


  <details><summary><b>DatePicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker.Root.click.chromium.png|412| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>